fahrschein/src/main/java/org/zalando/fahrschein/NakadiClient.java

```diff
@@ -134,7 +135,11 @@ public Subscription subscribe(String applicationName, Set<String> eventNames, St
      * @return
      * @throws IOException
      */
-    public Subscription subscribe(String applicationName, Set<String> eventNames, String consumerGroup, SubscriptionRequest.Position readFrom, Set<Cursor> initialCursors) throws IOException {
+    public Subscription subscribe(String applicationName, Set<String> eventNames, String consumerGroup, SubscriptionRequest.Position readFrom, List<Cursor> initialCursors) throws IOException {
+
+        if(readFrom.equals(SubscriptionRequest.Position.CURSORS)){
```

Comment:
Enum values should in general be compared using `==` instead of `equals`, that way the compiler can warn you if accidentally try to compare enums of different types.